web-api/openapi/front_office.yaml

```diff
@@ -60,9 +72,12 @@ paths:
     patch:
       tags:
         - front office
-      summary: Add interaction between a user and a document
+      summary: Adds an interaction between a user and a document.
       description: |-
-        The interaction is used to provide personalized documents to the user.
+        Use this method to register an interaction between a user and a document.
+        Currently, we only support a `Positive` interaction, which is equivalent to a user who "likes" a certain document.
+        For web sites, consider triggering this method whenever a certain document url loads.
```

Comment:
We don't want to get when the URL load, but if the user stays "long enough" or scroll a certain amount of the article. Something that could point out that the user might be interested in this and is not a misclick.

Reply:
They need to already show suggested articles when a user opens a web page,
so the flow is await updateInteractions (current document viewed)
then, get personalized content.

This is also why we went from 2 to 1 interaction, so we could do this immediately.

In the end, it's up to them to implement it of course